k8s/main.ts

```diff
@@ -11,9 +11,7 @@ export class MyChart extends PennLabsChart {
     const backendImage = 'pennlabs/penn-courses-backend';
     const secret = 'penn-courses';
 
-    new RedisApplication(this, 'redis', { deployment: { 
```

Comment:
@joyliu-q @alxkp can one of you guys review this change?

Reply:
resolved on call